docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+    server:
+        image: benetech/mathshare.server
+        environment:            
+            - SPRING_DATASOURCE_URL=jdbc:postgresql://postgresql:5432/mathshare
+        ports:
+            - 8080:8080
+        depends_on:
+            - postgresql
+    postgresql:
+        image: postgres:9.5.12
+        environment:
+        - POSTGRES_PASSWORD=mysecretpassword
+        - POSTGRES_USER=postgres
+        - POSTGRES_DB=mathshare
+        volumes:
+        - db-data:/var/lib/postgresql/data
```

Comment:
There is something wrong with indentations

Reply:
Right, I will fix it